basil.js

```diff
@@ -1494,44 +1494,56 @@ pub.doc = function(doc) {
 /**
  * Sets the size of the current document, if arguments are given.
  * If only one argument is given, both the width and the height are set to this value.
+ * Alternatively a string can be given as the only argument to apply an existing page size preset ("A4", "Letter" etc.).
```

Comment:
I don't think you can. I have looked around for a bit as well, and couldn't find anything. You can just catch an error to find out that a page size preset does not exist, afaik.